AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -298,7 +298,7 @@ msnbc.com,nbcnews.com###cx_bottom_banner
 pokemongolive.com##.nl-cpb
 openedition.org###privacynotice
 eartharxiv.org##div[class^="__cookie-banner__"]
-netenpiac.hu,profikisgep.hu,majorsoft.eu###firstLogNanobar
+pi-shop.hu,netenpiac.hu,profikisgep.hu,majorsoft.eu###firstLogNanobar
```

Comment:
Please hide the element with #$# rules (when scrolling down you have white bar at top).

```
pi-shop.hu#$##mobile-nav { top:0px!important; }
pi-shop.hu#$#body.aurora-nanobar-active { padding-top:0px!important; }
pi-shop.hu#$##firstLogNanobar { display:none!important; }
```

Reply:
plz don't add styles without spaces

Reply:
better `display: none !important;` instead of `display:none!important;`